src/utils_general/raster_manipulation.py

```diff
@@ -1,8 +1,9 @@
 import logging
 import geopandas as gpd
-import numpy as np
-from rasterstats import zonal_stats
 import pandas as pd
+import xarray as xr
+from typing import List
+import rioxarray  # noqa: F401
```

Comment:
Imports should be grouped into standard and external libraries, see [here](https://www.python.org/dev/peps/pep-0008/#imports)